client/extensions/woocommerce/state/sites/orders/utils.js

```diff
@@ -36,3 +34,27 @@ export function getSerializedOrdersQuery( query = {} ) {
 
 	return serializedQuery;
 }
+
+/**
+ * Remove temporary IDs used for adding products & fees to an existing order
+ * The IDs for items needs to be null when sent to the API for the remote site
+ * to correctly save them as new line items/fee items.
+ *
+ * @param  {Object} order  Order object
+ * @return {Object}        Order object, with no temporary IDs
+ */
+export function removeTemporaryIds( order ) {
+	const newOrder = { ...order };
+	for ( const type of [ 'line_items', 'fee_lines', 'coupon_lines', 'shipping_lines' ] ) {
+		if ( order[ type ] ) {
+			newOrder[ type ] = order[ type ].map( item => {
+				if ( ! isNumber( item.id ) ) {
+					return omit( item, 'id' );
+				}
+				return item;
+			} );
+		}
+	}
+
+	return newOrder;
```

Comment:
`return https://www.youtube.com/watch?v=9GMjH1nR0ds`

could not resist this time.